server/build_event_protocol/event_parser/event_parser.go

```diff
@@ -127,7 +127,7 @@ func NewStreamingEventParser() *StreamingEventParser {
 		startTimeMillis:        undefinedTimestamp,
 		endTimeMillis:          undefinedTimestamp,
 		screenWriter:           terminal.NewScreenWriter(),
-		allowedEnvVars:         []string{"USER", "GITHUB_ACTOR", "GITHUB_REPOSITORY", "GITHUB_SHA", "GITHUB_RUN_ID"},
+		allowedEnvVars:         []string{"USER", "GITHUB_ACTOR", "GITHUB_REPOSITORY", "GITHUB_SHA", "GITHUB_RUN_ID", "BUILDKITE_BUILD_URL"},
```

Comment:
Just a sanity check, is it possible for these to include auth tokens, e.g. `https://user:token@buildkite.whatever`?

Reply:
Doesn't seem like it, this particular environment variable seems to be made specifically for this purpose:
https://buildkite.com/docs/pipelines/environment-variables#bk-env-vars-buildkite-build-url